universe/interface.go

```diff
@@ -614,6 +614,22 @@ func (t ProofType) String() string {
 	return fmt.Sprintf("unknown(%v)", int(t))
 }
 
+// NewProofTypeFromString returns the proof type corresponding to the given
+// string.
+func NewProofTypeFromString(typeStr string) (ProofType, error) {
+	switch typeStr {
+	case "unspecified":
```

Comment:
IIUC, we never actually allow the "unspecified" type in the db. After all we need to know _where_ to put the proof.

Reply:
Yes that's true. `case "unspecified":` is just dead code right now. I included it to make `NewProofTypeFromString` complete.